tools/umi_tools/main.nf

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.preview.dsl = 2
+
+// Local default params
+params.internal_process_name = 'umitools_dedup'
+
+// Process definition
+process umitools_dedup {
+    publishDir "${params.outdir}/${params.internal_process_name}",
+        mode: "copy", overwrite: true
+
+    container 'luslab/nf-modules-umitools:latest'
+
+    input:
+      tuple val(sample_id), path(bai), path(bam)
+       
+    output:
+      tuple val(sample_id), path("*.dedup.bam"), emit: dedupBam
+      path "*.dedup.log", emit: report
+
+    shell:
+
+    // Init
+    internal_prog = "umi_tools dedup"
+    internal_args = "--log=${sample_id}.dedup.log"
+
+    // Check main args string exists and strip whitespace
+    if(params.umitools_dedup_args) {
+        ext_args = params.umitools_dedup_args
+        internal_args += " " + ext_args.trim()
+    }
+
+    // Contruct CL line
+    internal_cl = "${internal_prog} ${internal_args} -I $bam -S ${sample_id}.dedup.bam --output-stats=${sample_id}"
```

Comment:
```suggestion
    command = "${internal_prog} ${internal_args} -I $bam -S ${sample_id}.dedup.bam --output-stats=${sample_id}"
```

Reply:
As with https://github.com/nf-core/modules/pull/34/files#r444046887, why not just have this in the script block?

I guess the answer is the `println` for verbose mode, but I'm not sure that this adds a lot..? It's already logged in a load of places already.